Makefile

```diff
@@ -173,5 +171,8 @@ htmlproofer-docker-image:
 .PHONY: htmlproofer-docker-image
 
 clean:
-	rm -rf public/ resources/ node_modules/ upstream/ content/docs/user_guide/compatibility/linux/
+ifneq ("$(wildcard bazel_user_root/)","")
```

Comment:
Run `bazel clean`?

Reply:
I don't run bazel clean for a few reasons:
1. I'm running bazel in a Docker container and it seems a bit overkill to run bazel clean here since bazel_user_root is the only side effect.
2. bazel clean won't work if upstream/gvisor isn't present. I want make clean to be a noop if the project is already clean (I check for the existence of bazel_user_root here for the same reason)